nautobot/core/models/__init__.py

```diff
@@ -298,3 +299,24 @@ def natural_key_args_to_kwargs(cls, args):
                 f"expected no more than {len(natural_key_field_lookups)} but got {len(args)}."
             )
         return dict(zip(natural_key_field_lookups, args))
+
+
+class ContentTypeRelatedQuerySet(RestrictedQuerySet):
```

Comment:
Moved this from `name_color_content_types.py` to this file so as to permit importing and using it in the Cloud model definitions without a circular import error.